### ISSUE TITLE
Is there any way to disable standard webrepl access to the device?

### ISSUE THREAD

**Issue by chrisb2** [Carglglz/upydev#5]
Is there any way to disable standard webrepl access to the device, so that only SSLWebREPL can be used?

thanks,
Chris

**Comment by Carglglz** [comment]
@chrisb2 
Not exactly, since in SSLWebREPL mode the server is the host computer, and the client is the device, so you need a way to the tell the device to connect to the server and this is done through standard WebREPL access.

However I did a mode that enables WebSecureREPL, so connection is through SSL all the time.

> wssrepl : to enter the terminal WebSecureREPL; CTRL-x to exit, CTRL-d to do soft reset.
To see more keybindings info do CTRL-k. 
**REPL over WebSecureSockets (This needs use of 'sslgen_key -tfkey', 'update_upyutils' and enable WebSecureREPL in the device "import wss_repl;wss_repl.start(ssl=True)")**

So substitute the standard `webrepl.start()` in your code (in boot.py or main.py) with the line above
`import wss_repl;wss_repl.start(ssl=True)`
After WebSecureREPL is enabled you can access the device with `wssl`mode:

> wssl: to access ssl_wrepl if WebSecureREPL is enabled in a 'ssh' style command to be used like e.g.: "upydev wssl@192.168.1.42" or if a device is stored in a global group called "UPY_G" (this needs to be created first doing e.g. "upydev make_group -g -f UPY_G -devs foo_device 192.168.1.42 myfoopass") then the device can be accessed as "upydev wssl@foo_device"

However not all the upydev "standard" commands from the works with WebSecureREPL but I think 'get' and 'put' commands work as expected and maybe a few more (It's not hard to do but I haven't had the time...) 

If you have any more doubts or questions about this I'm glad to answer them 👍
